pkg/ddevapp/global_dotddev_assets/commands/web/pint

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+#ddev-generated
+## Description: Run Pint inside the web container
+## Usage: pint [flags] [args]
+## Example: "ddev pint --dirty" for fixing your current changes lint errors or "ddev pint --test app/Models" for only analyzing your models without autofix. 
+## ProjectTypes: laravel
+
+./vendor/bin/pint "$@"
```

Comment:
vendor/bin is already in the $PATH, so this can just be
```suggestion
pint "$@"
```

However, `pint` is not always available/installed, right? So you might want to do what the drush command does and tell people what to do,

https://github.com/ddev/ddev/blob/master/pkg/ddevapp/global_dotddev_assets/commands/web/drush#L10-L13

Reply:
> vendor/bin is already in the $PATH, so this can just be

Is this new? Didn't work for me with v1.22.7 when I initially tried it that way.

> However, pint is not always available/installed, right?

Yeah, wanted to try this on gitpod, but that required I create the PR first (and that's why I used a draft)
I will check what the last releases of laravel have, but probably the sanity check makes sense anyway (even if it's a default, I could remove the package on my project)

Reply:
> Is this new? Didn't work for me with v1.22.7 when I initially tried it that way.

vendor/bin has been in $PATH for years.

> Yeah, wanted to try this on gitpod, but that required I create the PR first (and that's why I used a draft)
I will check what the last releases of laravel have, but probably the sanity check makes sense anyway (even if it's a default, I could remove the package on my project)

You may like it better as an add-on, don't know.